scripts/lib/CIME/build.py

```diff
@@ -355,7 +355,7 @@ def _build_libraries(case, exeroot, sharedpath, caseroot, cimeroot, libroot, lid
 
     # Build shared code of CDEPS nuopc data models
     cdeps_build_script = None
-    if comp_interface == "nuopc" and not cmeps_driver or not 'nems' in cmeps_driver:
+    if comp_interface == "nuopc" and (not cmeps_driver or not 'nems' in cmeps_driver):
```

Comment:
Can you explain why you use `not 'nems' in cmeps_driver` instead of `cmeps_driver != 'nems'`?